force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -300,15 +387,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Map<String, Integer> addItemsToMap() {
         // Initialize an empty Map
-
+        Map<String, Integer> fruitQuantity = new Map<String, Integer>{
         // Add key-value pairs to the Map
-        // add Apples => 5
-        // add Oranges => 10
-        // add Bananas => 15
-        // add Pears => 20
-        // add Grapes => 25
-
-        return null; // Replace null with the variable you used to store the result
+        // add 
+        'Apples' => 5,
+        'Oranges' => 10,
+        'Bananas' => 15,
+        'Pears' => 20,
+        'Grapes' => 25
+        };
+        return fruitQuantity; // Replace null with the variable you used to store the result
     }
```

Comment:
Love the use of the Map constructor here. I would have just returned it directly rather than assigning it to a variable. good job